core/domain/user_services.py

```diff
@@ -201,6 +210,42 @@ def get_users_settings(user_ids):
     return result
 
 
+def generate_profile_picture(user_id):
```

Comment:
(Optional) Maybe call this generate_initial_profile_picture(), to make it more obvious that this is only meant to be used for first-time users.